pandas/core/arrays/datetimes.py

```diff
@@ -793,17 +793,24 @@ def month_name(self, locale=None):
         """
         Return the month names of the DateTimeIndex with specified locale.
 
+        .. versionadded:: 0.23.0
+
         Parameters
         ----------
-        locale : string, default None (English locale)
-            locale determining the language in which to return the month name
+        locale : str, optional
+            Locale determining the language in which to return the month name.
+            Default is English locale.
 
         Returns
         -------
-        month_names : Index
-            Index of month names
+        Index
+            Index of month names.
 
-        .. versionadded:: 0.23.0
+        Examples
+        --------
+        >>> idx = pd.DatetimeIndex(start='2018-09', freq='M', periods=3)
+        >>> idx.month_name()
+        Index(['September', 'October', 'November'], dtype='object')
```

Comment:
If you don't mind, can you first show `idx` (i.e. `>>> idx` after creating the index). Also, if we use '2018-01' it'd make the example more obvious.

Reply:
Sure, will change it and upload it later today. :+1:

About the locale discussion, I too think it must be a misunderstanding. With an average of 10 words per comment, I am having a hard time trying to understand what he tries to say. :sweat_smile: 

If you understand both of us, could you help us? :joy: